### PR TITLE
Fix Prev/Next buttons and teleportation

### PR DESCRIPTION
**Pull request by Bobcat00**
- Adds Prev/Next buttons to the near command output
- Doesn't show Next button if there's only one page of results
- Corrects a check for Spigot

### CODE REVIEW

File: src/main/java/me/botsko/prism/utils/MiscUtils.java [hunk 1]
```diff
@@ -162,15 +162,19 @@ public static void sendClickableTPRecord(ActionMessage a, CommandSender player){
 	public static void sendPageButtons(QueryResult results, CommandSender player){
 		if (player instanceof Player) {
 			if (PaperLib.isPaper()) {
-				if (results.getPage() == 1)
-					player.sendMessage(MiscUtils.getNextButton());
+	            if (results.getPage() == 1) {
```
Comment (addstar) [inline src/main/java/me/botsko/prism/utils/MiscUtils.java]: These are all great changes! :clap: Hope you don't mind me being pedantic and but can I ask you to fix the indentation? :grin:

Comment (Bobcat00) [reply src/main/java/me/botsko/prism/utils/MiscUtils.java]: Changed spaces to tabs

**Pull request merged**